src/webview/components/options-widget.tsx

```diff
@@ -119,6 +121,13 @@ export class OptionsWidget extends React.Component<OptionsWidgetProps, {}> {
 
     override render(): React.ReactNode {
         this.formConfig.initialValues = this.optionsFormValues;
+        const memoryReadArgumentHint = (userValue: string | number, memoryValue?: string | number): ReactNode | undefined => {
+            if (memoryValue && userValue !== memoryValue) {
+                return <small className="form-options-memory-read-argument-hint">Actual: {memoryValue}</small>;
+            }
+
+            return;
```

Comment:
I think this return isn't needed, right? If you prefer to keep it, I'd make it explicit to `return undefined;`.